source/camera/CullingManager.cpp

```diff
@@ -0,0 +1,253 @@
+/*
+ *  Copyright (C) 2011-2015  OpenDungeons Team
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <OgreVector3.h>
+#include <OgreCamera.h>
+
+#include "camera/CullingManager.h"
+#include "camera/CameraManager.h"
+#include "gamemap/GameMap.h"
+
+#include "entities/Creature.h"
+#include "utils/VectorInt64.h"
+#include "utils/LogManager.h"
+#include "render/RenderManager.h"
+#include <sstream>
+#include <algorithm>
+
+//! Values used to know whether to show and/or hide a mesh
+static const int HIDE =  1;
```

Comment:
This should be static in the CullingManager class to avoid problems with same constant name (pretty likely since names are very common)